cratedb_toolkit/io/iceberg.py

```
@@ -0,0 +1,231 @@
+        "file+iceberg://./var/lib/iceberg/?catalog=default&namespace=demo&table=taxi_dataset"
+
+    # Save to AWS S3.
+    ctk save table \
+        --cluster-url="crate://crate@localhost:4200/demo/taxi-tiny" \
+        "s3+iceberg://bucket1/?catalog=default&namespace=demo&table=taxi-tiny&s3.access-key-id=<your_access_key_id>&s3.secret-access-key=<your_secret_access_key>&s3.endpoint=<endpoint_url>&s3.region=<s3-region>"
+    """  # noqa:E501
+
+    cratedb_address = DatabaseAddress.from_string(source_url)
+    cratedb_url, cratedb_table = cratedb_address.decode()
+    if cratedb_table.table is None:
+        raise ValueError("Table name is missing. Please adjust CrateDB database URL.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)